include/cellular_config_defaults.h

```diff
@@ -331,8 +331,8 @@
  * <b>Possible values:</b>`0 or 1`<br>
  * <b>Default value (if undefined):</b> 0
  */
-#ifndef CELLULAR_CONFIG_STATIC_ALLOCATION_COMM_CONTEXT
```

Comment:
Same as above reason.

Reply:
Currently this change is for the the warning from MISRA, I agree to create another PR for the purpose of removing.